Kfilter/src/main/java/com/isaacudy/kfilter/KfilterView.kt

```diff
@@ -458,11 +489,22 @@ class KfilterView @JvmOverloads constructor(context: Context,
         }
 
         override fun onFling(e1: MotionEvent, e2: MotionEvent, velocityX: Float, velocityY: Float): Boolean {
+            val yDistance: Float = Math.abs(e1.getY() - e2.getY())
+            val velocityY1 = Math.abs(velocityY)
+            if (velocityY1 > 400 && yDistance > 400) {
```

Comment:
What is the "400" number based on?

Reply:
I just check user swiping up or down on the screen so return this function if velocity is greater then 400